string command
string featureid
string btime
string bmode
